.gitignore

```diff
@@ -2,3 +2,4 @@
 __pycache__
 .DS_Store
 /settings.json
+.vscode/
```

Comment:
Please remove the change to suppress the `.vscode` folder as we have project specific editor settings there.

Reply:
Will do